src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8472,6 +8473,87 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/linked-rules':
+    post:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: Link rules from another folder
+      desciption: |
+        **Note:** this endpoint is available in Alfresco 7.3 and newer versions.
+        
+        Links the rules from another folder to the folder node **nodeId**.
+      operationId: linkRules
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#parameters/nodeIdParam'
+        - in: body
+          name: linkedRulesBodyCreate
+          description: The ruleset to add.
+          required: true
+          schema:
+            $ref: '#/definitions/LinkedRulesBodyCreate'
+      responses:
+        '201':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/LinkedRules',
+        '400':
+          description: |
+            Invalid parameter: **folderId** or **linkedRulesBodyCreate** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: User does not have permission to create a link to this folder
+        '404':
+          description: |
+            **nodeId** or **folderId** does not exist
+        '409':
+          description: Cannot link folder rules to itself
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/linked-rules/{linkedFolderId}':
+    delete:
```

Comment:
Is "linked-rules" an entity or an operation ?

If operation then we don't need {linkedFolderId} ?

If entity, I think it should be called "rulesets" as per parallel [PR](https://github.com/Alfresco/rest-api-explorer/pull/161/files) ?